app/jobs/characterize_job.rb

```diff
@@ -1,34 +1,10 @@
+# Characterizes the file at 'filepath' if available, otherwise, pulls a copy from the repository
+# and runs characterization on that file.
+
```

Comment:
A trivial thing: would you consider dropping the newline between the class documentation and the `class` declaration?